extension/bearertokenauthextension/bearertokenauth.go

```diff
@@ -22,13 +22,15 @@ import (
 var _ credentials.PerRPCCredentials = (*PerRPCAuth)(nil)
 
 // PerRPCAuth is a gRPC credentials.PerRPCCredentials implementation that returns an 'authorization' header.
+// NOTE: Needs a function instead of static values so the data can be refreshed
```

Comment:
Is this note better as an issue?

Reply:
I can try and reword the note - this is merely here to help people not trip up on the bug that was originally here as the design does not make it obvious that you actually need a function pointer rather than a static value (that then will not get refreshed)